Library/Formula/valgrind.rb

```diff
@@ -3,18 +3,11 @@ class Valgrind < Formula
   homepage "http://www.valgrind.org/"
 
   stable do
-    url "http://valgrind.org/downloads/valgrind-3.10.1.tar.bz2"
-    sha256 "fa253dc26ddb661b6269df58144eff607ea3f76a9bcfe574b0c7726e1dfcb997"
-
-    # Look for headers in the SDK on Xcode-only systems: https://bugs.kde.org/show_bug.cgi?id=295084
-    # Fix duplicate symbols error on Lion: https://bugs.kde.org/show_bug.cgi?id=307415
-    patch do
-      url "https://gist.githubusercontent.com/jacknagel/369bedc191e0a0795358/raw/a71e6c0fdcb786fdfde2fc33d71d555b18bcfe8d/valgrind-sdk-paths-Makefile-in.diff"
-      sha256 "ff8e8186af6985f9ffb8dca8fbea944e807d1caf144f320dca0f995d0417db7f"
-    end
+    url "http://valgrind.org/downloads/valgrind-3.11.0.tar.bz2"
+    sha256 "6c396271a8c1ddd5a6fb9abe714ea1e8a86fce85b30ab26b4266aeb4c2413b42"
 
     # Revisit the below requirement with each release
-    depends_on MaximumMacOSRequirement => :mavericks
+    depends_on MaximumMacOSRequirement => :el_capitan
```

Comment:
Since there's no known OS X version above El Cap at the moment, we can just kill this line.